lib/rules/no-effect-override.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @fileoverview Rule to prevent (accidental) override of redux-saga effects with scoped function
+ * @author Philipp Kursawe
+ */
+/** eslint-disable semi */
+"use strict"
+
+/**
+ * List all variable in a given scope
+ *
+ * Contain a patch for babel-eslint to avoid https://github.com/babel/babel-eslint/issues/21
```

Comment:
A link to the code where it was copied from won't hurt.
https://github.com/yannickcr/eslint-plugin-react/blob/442d20b6e1bb5daeacd5234f253f963af90cc3e4/lib/rules/react-in-jsx-scope.js#L22-L23

The referenced bug was closed. I'm not sure if there is a built-in utility function in eslint to perform that variable lookup. If there is, probably it should be used instead.